docs/user/dashboard/aggregation-reference.asciidoc

```diff
@@ -0,0 +1,222 @@
+[[aggregation-reference]]
+== Aggregation reference
+
+{kib} supports many types of {ref}/search-aggregations.html[{es} aggregations] that you can use to build complex summaries of your data. 
+
+By using a series of {es} aggregations to extract and process your data, you can create panels that tell a 
+story about the trends, patterns, and outliers in your data.
+
+[float]
+[[bucket-aggregations]]
+=== Bucket aggregations
+
+For information about Elasticsearch bucket aggregations, refer to {ref}/search-aggregations-bucket.html[Bucket aggregations].
+
+[options="header"]
+|===
+
+| Type | Visualizations | Data table | Markdown | Lens | TSVB 
+
+| Histogram
+^| X
+^| X
+^| X
+| 
+| 
+
+| Date histogram
+^| X
+^| X
+^| X
+^| X
+^| X
+
+| Date range
+^| X
+^| X
+^| X
+| 
+| 
+
+| Filter
+^| X
+^| X
+^| X
+| 
+^| X
+
+| Filters
+^| X
+^| X
+^| X
+| 
+^| X
+
+| GeoHash grid
+^| X
+^| X
+^| X
+| 
+| 
+
+| IP range
+^| X
+^| X
+^| X
+| 
+| 
+
+| Range
+^| X
+^| X
+^| X
+| 
+| 
+
+| Terms
+^| X
+^| X
+^| X
+^| X
+^| X
+
+| Significant terms
+^| X
+^| X
+^| X
+^| X
+^| X
+
+|===
+
+[float]
+[[metrics-aggregations]]
+=== Metrics aggregations
+
+For information about Elasticsearch metrics aggregations, refer to {ref}/search-aggregations-metrics.html[Metrics aggregations].
+
+[options="header"]
+|===
+
+| Type | Visualizations | Data table | Markdown | Lens | TSVB
+
+| Average and Sum 
+^| X
+^| X
+^| X
+^| X
+^| X
+
+| Cardinality
+^| X
+^| X
+^| X
+^| X
+^| X
+
+| Max and Min
+^| X
+^| X
+^| X
+^| X
+^| X
+
+| Percentiles
```

Comment:
Median is a special type of percentile, it's not listed here though.

Reply:
I don't have that information. Where can I find it?